lib/pages/chat/chat_input_row_style.dart

```diff
@@ -1,5 +1,35 @@
+import 'package:fluffychat/di/global/get_it_initializer.dart';
+import 'package:fluffychat/utils/responsive/responsive_utils.dart';
+import 'package:flutter/material.dart';
+
 class ChatInputRowStyle {
+  static final ResponsiveUtils responsiveUtils = getIt.get<ResponsiveUtils>();
+
   static double get chatInputRowWidth => 52.0;
-  static double get chatInputRowHeight => 56.0;
-  static double get sendIconButtonSize => 34.0;
+  static double get chatInputRowHeightWeb => 56.0;
```

Comment:
```suggestion
  static const double chatInputRowHeightWeb = 56.0;
```
using const to improve render time